local_dev/start_containerd.sh

```diff
@@ -2,55 +2,34 @@
 # abort if any of the following commands fails or variables are undefined
 set -eu
 
-KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.15.0}"
-CALICO_VERSION="${CALICO_VERSION:-v3.8}"
+KUBERNETES_VERSION="${KUBERNETES_VERSION:-stable}"
 
-# Setup minikube
+# Setup minikube, requires minikube >= 1.12.1
 minikube delete
 minikube config set embed-certs true
 minikube start \
     --memory 4096 \
-    --cpus 4 \
-    --network-plugin=cni \
+    --cpus 2 \
+    --cni=calico \
     --container-runtime=containerd \
-    --extra-config=kubelet.network-plugin=cni \
-    --extra-config=kubelet.pod-cidr=192.168.0.0/16 \
-    --extra-config=controller-manager.allocate-node-cidrs=true \
-    --extra-config=controller-manager.cluster-cidr=192.168.0.0/16 \
     --bootstrapper=kubeadm \
     --host-only-cidr=172.17.17.1/24 \
-    --insecure-registry=localhost:5000 \
     --kubernetes-version="${KUBERNETES_VERSION}"
 
-# Setup the minikube docker registry and calico
```

Comment:
The docker registry is still setup by minikube so the comment is still valid? At least the first part.